src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form">
+      <label
+        for="big-bar"
+        class="form__label-big"
+      ></label>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="form__bar form__bar--big"
+        id="big-bar"
+      />
+
+      <label
+        for="small-bar"
+        class="form__label-small"
+      ></label>
+
+      <input
+        type="text"
+        data-qa="small"
+        placeholder="Try “Los Angeles“"
+        class="form__bar form__bar--small"
+        id="small-bar"
+      />
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. But don't add them between parent and child elements. In this case, there should be an empty line between the `label` and `input` elements, but not between the `form` and the first `label`.